flash/image/classification/integrations/baal/__init__.py

```diff
@@ -0,0 +1,31 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import flash
+from flash.core.utilities.imports import requires
+from flash.image.classification.integrations.baal.loop import ActiveLearningDataModule, ActiveLearningLoop  # noqa F401
+
+
+class ActiveLearningTrainer(flash.Trainer):
```

Comment:
is this just here to make the example shorter or are you saying that loop customization should be packaged like this in general?

Reply:
That's a great and open question. I think it is simpler to package it this way in Flash to make example shorter.

Reply:
I strongly think we should not do it that way! Whatever we do here is likely to be used by others as well. I think we need to introduce a formal API for exchanging loops. IMO if we need to subclass the trainer (which is supposed to be the fundamental element of PL) for something we also want to support in PL in general, it shows that we might need a better API there (or in this case: An API at all).

Reply:
Fine with me ! Let me make it explicit within the example how we connect loops.